manifest.md

```diff
@@ -85,13 +85,20 @@ A client will distinguish a manifest list from an image manifest based on the Co
 
     This OPTIONAL property contains arbitrary metadata for the manifest list.
     Annotations is a key-value, unordered hashmap.
-    Keys are unique, and best practice is to namespace the keys.
+    Keys MUST be unique, and best practice is to namespace the keys.
+	Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
```

Comment:
that ought to be clarified. I had in mind that it would be namespaced more
like an URL, rather than a java package.

On Thu, Jul 7, 2016 at 6:44 PM, Stephen Day <notifications@github.com>
wrote:

> In manifest.md
> <https://github.com/opencontainers/image-spec/pull/164#discussion_r70000997>
> :
>
> > @@ -85,13 +85,20 @@ A client will distinguish a manifest list from an image manifest based on the Co
> >
> >      This OPTIONAL property contains arbitrary metadata for the manifest list.
> >      Annotations is a key-value, unordered hashmap.
> > -    Keys are unique, and best practice is to namespace the keys.
> > +    Keys MUST be unique, and best practice is to namespace the keys.
> > +	Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
>
> Is this not org.opencontainers or is that not how it is done here?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/164/files/e969158a0e4371760a72da3492b1b6803b03dc90#r70000997>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAEF6dv8Fi4ZqyfxK79cBiFMdZTamp01ks5qTYFjgaJpZM4I-n6z>
> .
>